CMakeLists.txt

```diff
@@ -505,13 +505,24 @@ add_dependencies(zig zig_build_zig1)
 
 install(TARGETS zig DESTINATION bin)
 
-set(ZIG_INSTALL_ARGS "build"
-    --override-lib-dir "${CMAKE_SOURCE_DIR}/lib"
-    "-Dlib-files-only"
-    --prefix "${CMAKE_INSTALL_PREFIX}"
-    "-Dconfig_h=${ZIG_CONFIG_H_OUT}"
-    install
-)
+if(NOT SYSTEM_LINKER_HACK)
```

Comment:
I don't think we need any cmake changes here. This command here only installs lib files and would not make any use of that flag.